src/clojure/nrepl/cmdline.clj

```diff
@@ -68,6 +68,15 @@
   (some (partial = input)
         ['exit 'quit '(exit) '(quit)]))
 
+(defn print-repl-intro
```

Comment:
I think this should be a function that generates simply a string and when it's needed the message would be one `println` away.